rdmo/management/management/commands/merge_attributes.py

```diff
@@ -118,23 +118,35 @@ def handle(self, *args, **options):
                 self.stdout.write(affected_instances_msg)
 
 
-def get_valid_attribute(attribute, message_name=''):
+def get_attribute_from_uri(attribute_uri, message_name=None):
+    try:
+        attribute = Attribute.objects.get(uri=attribute_uri)
+        return attribute
+    except Attribute.DoesNotExist as e:
+        raise CommandError(f"{message_name} attribute {attribute_uri} does not exist.") from e
+    except Attribute.MultipleObjectsReturned as e:
+        raise CommandError(f"{message_name} attribute {attribute_uri} returns multiple objects.") from e
+
+
+def validate_attribute(attribute, message_name=None, allow_descendants=None):
+    if not isinstance(attribute, Attribute):
+        raise CommandError(f"{message_name} attribute argument should be of type Attribute.")
+
+    if not allow_descendants:
```

Comment:
combined with this

Reply:
The two ifs could be one line :stuck_out_tongue: